### ISSUE TITLE
Label Height Larger Than 128 px / 18mm

### ISSUE THREAD

**Issue by sonaro** [talpadk/ptouch-php#1]
Hello,

Have you had any luck printing a larger label height?  I am attempting to print to a 24mm label.

Thank you kindly!

**Comment by talpadk** [comment]
The label cartridge I have is the 24mm one that came with the printer.
What kind of issues are you having with the printouts?

**Comment by sonaro** [comment]
I was not able to print the full label height of 24mm.  It appears the print head is only 18mm wide, and therefore purchasing a label larger than 18mm has no advantage with the PT-P700.  Can you confirm this?

**Comment by talpadk** [comment]
The 24mm role has 128 printable pixels, at the printer resolution of 180 DPI that should result in a 18.06 mm high image.
Just like regular printers the label writer has some margins on which the printer cannot print.

Smaller rolls have less than 128 pixels of printable space:
3.5 mm -> 24px ~= 3.39mm
6 mm -> 32px ~= 4.52mm
9 mm -> 50px ~= 7.06mm
12 mm -> 70px ~= 9.88mm
18 mm -> 112px ~= 15.80mm
24 mm ->128px ~=18.06 mm

In other words there are some benefits of the larger labels.

**Comment by talpadk** [comment]
P.S. These values are for the TZe tapes, the heat shrink ones have slightly different print areas.

The information on the print height is from cv_pth500p700e500_eng_raster_110.pdf page 25 and 26 (pdf numbering, not the page numbers on the page fotter)

**Comment by sonaro** [comment]
Thanks for referencing the Software Developer's PDF.  I was not aware this document existed and the guide is extremely helpful!

It appears the physical width of the print head for the PT-P700 is 18mm/128px, as mentioned in the Specifications [here](http://www.brother-usa.com/Labeling_Solutions/ModelDetail/7/PTP700/spec) as "Max. Print Height".  It appears the specification is incorrectly listed on the website in the "Software Features" section, rather than the "Hardware Features" section.

I am currently able to print with 0px margin (edge to edge) on the 18mm cartridges.  I was attempting to print with 0px margin on the 24mm cartridge, but was unsuccessful since it appears the physical width of the print head of the PT-P700 is only 18mm/128px.  Printing edge-to-edge is important for my application since I am stacking multiple labels together to attain a wider print area (2x18mm), and need a seamless transition between labels.

For reference, other Brother label printers listed below have a print head physically wide enough to print on the 24mm cartridges with 0px margin, although I have not tested them for edge-to-edge support yet.
[PT-9700PC / PT-9800PCN](https://www.brother-usa.com/VirData/Content/en-US/PTouch/Consumer/UsersManual/UM_PT_9700PC_9800PCN_EN_2726.PDF), p.49, Maximum printing width = 1.07" (27.1 mm), [ESC/P Dev Manual](http://download.brother.com/welcome/docp000584/cv_pt9700_eng_escp_103.pdf)
[PT-P900W / PT-P950NW](http://download.brother.com/welcome/docp100201/cv_ptp950nw_useng_usr_a.pdf), p.170,  Maximum printing width = 1.26" (32 mm), [Raster Dev Manual](http://download.brother.com/welcome/docp100187/cv_ptp900_eng_ptemp_101.pdf)

[Label Printer Development / Command Support Map](http://download.brother.com/pub/com/dev/pdf/label_printer_dev_map.pdf)

I will keep the PT-P700 for my application since the printer is much less expensive than the other printers that have a larger print head and I also do not need the higher DPI.
